raiden_installer/ethereum_rpc.py

```diff
@@ -12,8 +12,8 @@
 def make_web3_provider(url: str, account: Account) -> Web3:
     w3 = Web3(HTTPProvider(url))
     w3.eth.setGasPriceStrategy(fast_gas_price_strategy)
-    w3.middleware_stack.add(construct_sign_and_send_raw_middleware(account.private_key))
-    w3.middleware_stack.inject(geth_poa_middleware, layer=0)
+    w3.middleware_onion.add(construct_sign_and_send_raw_middleware(account.private_key))
```

Comment:
Is this the same change that was done in the other Raiden modules when upgrading web3?

Reply:
No clue, what other Raiden modules?
I guess so since `middleware_stack` seems to be depreciated. 
Do you know who updated those other Raiden modules? If so could you ask them if it's the same change?